x/incentive/README.md

```diff
@@ -0,0 +1,301 @@
+# Incentive
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [States](#2-states)
+    1. [Parameters](#21-parameters)
+    2. [Gauge](#22-gauge)
+    3. [Reward Gauge](#23-reward-gauge)
+    4. [FinalityProviderHistoricalRewards](#24-finalityproviderhistoricalrewards)
+    5. [FinalityProviderCurrentRewards](#25-finalityprovidercurrentrewards)
+    6. [BTCDelegationRewardsTracker](#26-btcdelegationrewardstracker)
+3. [Messages](#3-messages)
+    1. [MsgUpdateParams](#31-msgupdateparams)
+    2. [MsgSetWithdrawAddress](#32-msgsetwithdrawaddress)
+    3. [MsgWithdrawReward](#33-msgwithdrawreward)
+4. [BeginBlocker](#4-beginblocker)
+5. [Queries](#5-queries)
+
+## 1. Introduction
+
+The Incentive module is responsible for determining the eligibility of rewards
+for both finality providers and BTC stakers. Initially, rewards are calculated
+based on the voting power of the finality provider, with the remaining rewards
+distributed to BTC delegations according to their voting power share.
```

Comment:
More precisely, rewards are distributed to BTC delegators but not BTC delegations